### DISCUSSION TITLE
`Expat error` when using `session.tables.schema`

### DISCUSSION THREAD

**Discussion by dee1603** [atoti/atoti#522] [Q&A] [answered]
Why am i receiving below error while running session.tables.schema? 
I'm thinking maybe there is issue with the graphviz module.

```
ExpatError: no element found: line 1, column 0
```

**Comment by Acriss** [comment, marked as answer]
Hey!
I'm sorry you're having this inconvenience.
Can you double check [here](https://gitter.im/atoti/atoti?at=60ece4bd67b72e1bfe0acfa5) , with this old discussion, that you're seeing the same error we got at that point in time ?
If so, can you confirm your graphviz installation is complete and correct ?

Cheers

**Reply by dee1603** [reply]
I think there is issue with its installation. It has to do with the path, environment variable. I do not have any admin rights for this.

**Reply by dee1603** [reply]
And yes it is the same issue as in the thread you mentionned.

**Reply by Acriss** [reply]
Alright ! I hope you were able to sort this installation issue.
Let me know if I can help!
Cheers
